docs/source/algorithms/ConvertPeaksWorkspace-v1.rst

```diff
@@ -0,0 +1,56 @@
+.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+This algorithm provides bi-directional conversion bewteen regular PeaksWorkspace and LeanElasticPeaksWorkspace.
+
+When casting a PeaksWorkspace into a LeanElasticPeaksWorkspace, information related to detector
+(DetID, BankName, Row, Col) is discarded.
+However, other experiment information are retained in the LeanElasticPeaksWorkspace, including
+run, hkl, goniometer, sample, as well as the corresponding information at the invidual Peak level.
+
+When casting a LeanElasticPeaksWorkspace into a PeaksWorkspace, a donor PeaksWorkspace with proper
+instrument configuration is required.
+Incorrect goniometer setting will lead to a Peak with negative wavelength, which will be excluded from the output
+PeaksWorkspace.
+
+Usage
+-----
+
+1) PeaksWorkspace to LeanElasticPeaksWorkspace
+
+.. code-block:: python
+
+    # import mantid algorithms, numpy and matplotlib
+    from mantid.simpleapi import *
```

Comment:
You don't need to include the imports in the usage examples.

Reply:
I thought these are sample codes that users can simply copy-and-paste.  I will remove these import statements now.